developer_setup.md

```diff
@@ -37,8 +37,17 @@
   sudo passwd postgres
   # ↑ will ask to choose a password
   sudo postgresql-setup initdb
+  # if the above usage of postgresql-setup fails with 'command not found', use the following command with the relevant postgres 9.x version - below an example for postgres 9.1:
```

Comment:
@carbonin I don't think it has to do with rhel vs fedora or related to systemd.
As described here in the postgres setup wiki, postgresql-setup is not used:
https://wiki.postgresql.org/wiki/YUM_Installation
I also asked a postgres DBA who told me postgresql-setup command was deprecated/removed.